src/components/LineChartComponent.tsx

```diff
@@ -1,20 +1,43 @@
 import * as React from 'react';
 import { LineChart } from '@mui/x-charts/LineChart';
 import { useTheme } from '@mui/material/styles';
+import Card from '@mui/material/Card/Card';
+import CardContent from '@mui/material/CardContent/CardContent';
+import Typography from '@mui/material/Typography';
 
-export default function LineChartComponent() {
+export type LineChartData = {
+  id: number
+  value: number
+  label: string
+}
+
+export interface LineChartComponentProps{
+  xAxis: [{}] //e.g. xAxis={[{data: [1, 2, 3, 5, 8, 10]}]}
+  yAxis: number[]
+  width?: number  
+  height?: number
+  title?: string
+}
+
+export default function LineChartComponent({xAxis, yAxis, title, width, height}: LineChartComponentProps) {
   const theme = useTheme()
   return (
-    <LineChart
-      xAxis={[{ data: [1, 2, 3, 5, 8, 10] }]}
-      series={[
-        {
-          data: [2, 5.5, 2, 8.5, 1.5, 5],
-          color: theme.palette.secondary.main
-        },
-      ]}
-      width={500}
-      height={300}
-    />
+    <Card sx={{ minWidth: 275 }}>
+      <CardContent>
+        <Typography sx={{ fontSize: 20 }} color="text.secondary" gutterBottom>
```

Comment:
Similar to pie chart, worth making typography a separate component (especially this repeats across both pie and line chart!)
And similarly, I think we are to chuck the component into the existing dashboard card, so would have to remove the card wrapper